spec/features/choose_organization_spec.rb

```diff
@@ -2,10 +2,12 @@
 
 feature 'Choose organization Flow' do
 
-  let(:user) { create(:user,
-                      metadata: Mumukit::Auth::Metadata.new( atheneum: {permissions: 'pdep/*'}))}
-  let(:user2) { create(:user,
-                      metadata: Mumukit::Auth::Metadata.new( atheneum: {permissions: ''}))}
+  let(:user) { create(:user)}
+  let(:user2) { create(:user)}
+  before do
+    user.set_permissions! student: 'pdep/_'
```

Comment:
Warning! Thw permiasion is pdep/* as before. The _ is a pattern that means 'do not care'